bin/main

```diff
@@ -56,40 +61,20 @@ def player_turn(arr, name, symbol, remaining_slots, player_arr)
   end
 end
 
-until tie
-  design_board(slots)
-
-  remaining_slots = slots.select { |item| item.is_a? Integer }
-
-  player_turn(slots, player1.name, 'X', remaining_slots, player1_arr)
-  sleep 1
-
-  if check_win(player1_arr, winning_numbers)
-    puts "#{player1.name.capitalize} wins"
-    break
-  end
-
-  if slots.none? { |item| item.is_a? Integer }
-    puts "It's a Tie!!"
-    break
-  end
-
-  remaining_slots = slots.select { |item| item.is_a? Integer }
-
-  design_board(slots)
-
-  player_turn(slots, player2.name, 'O', remaining_slots, player2_arr)
-
-  remaining_slots = slots.select { |item| item.is_a? Integer }
-  sleep 1
-
-  if check_win(player2_arr, winning_numbers)
-    puts "#{player2.name.capitalize} wins"
-    break
-  end
-
-  if slots.none? { |item| item.is_a? Integer }
-    puts "It's a Tie!!"
-    break
+class Board
```

Comment:
Class definition is prohibited in the project's requirements. Please move this class definition to another file.